### PR TITLE
CSS homepage

### PR DESCRIPTION
**Pull request by Brayan1425**
i added link of the css homepage

### CODE REVIEW

**Review by Theweeksad — CHANGES_REQUESTED** [review]
you have two CSS home page folders, please delete one folder, only the folder that has no code in it

**Review by Theweeksad — APPROVED** [review]
I see that everything is fine

File: css/homepage.css [hunk 1]
```diff
@@ -1,14 +1,16 @@
 .header__title {
-    text-align: center;
-
+    display: flex;
+    justify-content: center;
+    margin: 80px 0px 40px 0px;
```
Comment (johanmendezb) [inline css/homepage.css]: Please avoid margin top unless is the page/main container margin

File: css/homepage.css [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 }
 
 .header__subtitle {
-    margin-left: 30px;
+    margin: 30px;
```
Comment (johanmendezb) [inline css/homepage.css]: please use only margin bottom

File: css/homepage.css [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
 }
 
 .section__paragraph {
-    margin-left: 30px;
+    margin: 35px 35px 40px 30px;
```
Comment (johanmendezb) [inline css/homepage.css]: Please Avoid margin top unless is the page/main container margins

File: css/homepage.css [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
+    height: 260px;
```
Comment (johanmendezb) [inline css/homepage.css]: Avoid giving fixed values for section's height.

**Review by johanmendezb — CHANGES_REQUESTED** [review]
Please check the comments, and avoid repeating code specially for margins, add those in the main container

**Comment by devdevsamuel** [comment]
Please attach screenshots of the layout

File: css/global-styles.css [hunk 5]
```diff
@@ -1,25 +1,31 @@
...
 }
-
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+html{
+    font-size:16px;
+}
```
Comment (devdevsamuel) [inline css/global-styles.css:11]: pls use relative measurements

**Review by devdevsamuel — APPROVED** [review]

File: css/global-styles.css (hunk 5 above)
Comment (devdevsamuel) [reply css/global-styles.css:11]: and attach  ss!!  :(

**Pull request merged**